INSTALL.md

```diff
@@ -1,8 +1,16 @@
-### Installation instructions 
+## Installation instructions 
 
 Please follow the following steps in order to set up your development
 environment.
 
+### Docker
+
+1. Make sure [docker](https://www.docker.com/) is installed.
+2. Run `docker build -t bench-routes .`
+3. Run `docker run -p 9090:9090 -it bench-routes`
+
```

Comment:
please mention `superuser permission might be required`

Reply:
I've updated to specify the user needs to be in the docker group in order to run these commands.